cves/kernel/CVE-2016-6130.yml

```diff
@@ -305,10 +301,10 @@ discussion:
 
     Put any links to disagreements you found in the notes section, or any other
     comment you want to make.
-  discussed_as_security:
-  any_discussion:
-  note:
-vouch:
+  discussed_as_security: false
+  any_discussion: false
+  note: Nothing to add
```

Comment:
Maybe add where you looked that you were unable to find anything.

Reply:
Added where I looked! Did the same for an identical case below.